core/src/main/scala/org/apache/spark/ui/WebUI.scala

```diff
@@ -88,41 +90,41 @@ private[spark] abstract class WebUI(
     handlers += renderHandler
   }
 
-  /** Attach a handler to this UI. */
+  /** Attaches a handler to this UI. */
   def attachHandler(handler: ServletContextHandler) {
     handlers += handler
     serverInfo.foreach(_.addHandler(handler))
   }
 
-  /** Detach a handler from this UI. */
+  /** Detaches a handler from this UI. */
   def detachHandler(handler: ServletContextHandler) {
     handlers -= handler
     serverInfo.foreach(_.removeHandler(handler))
   }
 
   /**
-   * Add a handler for static content.
+   * Adds a handler for static content.
    *
    * @param resourceBase Root of where to find resources to serve.
    * @param path Path in UI where to mount the resources.
    */
-  def addStaticHandler(resourceBase: String, path: String): Unit = {
+  def addStaticHandler(resourceBase: String, path: String = "/static"): Unit = {
     attachHandler(JettyUtils.createStaticHandler(resourceBase, path))
   }
 
   /**
-   * Remove a static content handler.
+   * Removes a static content handler.
    *
    * @param path Path in UI to unmount.
    */
   def removeStaticHandler(path: String): Unit = {
```

Comment:
This method name is misleading. It will remove any handler, not just "static" handlers... and since you're touching this code...

Reply:
OK...since @vanzin requested I'm gonna make all the other changes while at it :)

Reply:
I think this could still be addressed? I think it's fine to rename the method as it's all private to Spark.